packages/bullet-list/example/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "../bullet-list.scss";
+
+const App = () => (
+    <>
+        <ul className="jkl-bullet-list">
+            <li className="jkl-bullet-list__item">Oh hi Mark</li>
+            <li className="jkl-bullet-list__item">Oh hi Johnny</li>
+            <li className="jkl-bullet-list__item">How is your life</li>
+            <li className="jkl-bullet-list__item--nested">Oh, yeah, nice</li>
```

Comment:
This is not semantically correct. We should try to find a way to nest properly. Maybe something like:
```scss
.jkl-bullet-list {
    & > &__item {
        // main style
    }
    & & > &__item {
        // nested style
    }
    & & & > &__item {
        // even more nested style
    }
}
```
Then we could do normal nesting like so
```html
<ul class="jkl-bullet-list">
    <li class="jkl-bullet-list__item">something
        <ul class="jkl-bullet-list">
            <li class="jkl-bullet-list__item">something else</li>
        </ul>
    </li>
</ul>
```

Reply:
Since you've implemented this change the example has to be updated.